sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/samples/HelloWorldSamples.cs

```diff
@@ -307,5 +296,10 @@ public void HelloWorld()
 
             #endregion
         }
+
+        private class LedgerEntry
```

Comment:
Is this to support the ability to deal with missing properties?

Reply:
It is to enable the cast for the null-check, per https://github.com/Azure/azure-sdk-for-net/pull/33063#discussion_r1089532143

I'm not 100% sold on this as the best API for this, given the need to create a type just to do a null check.  I hope to work on that API this week, so watch this space for changes to this.  In an earlier version, we were able to just check for null:

https://github.com/Azure/azure-sdk-for-net/pull/32501/files#diff-d55b2105899a17c3fafc1a079b6bb5a777c542671abad5f96c12223977a26597R188

I'll look at whether we can do that here when I get in there.